wordpress-mobile/WordPress-Android
# [Reader IA] Subscription filter chip group

Fixes #19606
Fixes #19611

This adds the UI part of the Subscription Filters chips/pills, both for Tags and Blogs. The content and behavior are just placeholders for now, and it's there mainly to review from a UI/Design perspective as well (specs, animations, etc).

Even so, the code for the TopBar Filter group should be final or only require minor changes when integrated with the business logic in the future.

Demo:

https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/28f72243-6ddd-4743-bfdc-628811c77b18

-----

## To Test:

1. Install and log into the Jetpack app
2. Go to Reader
3. Play around with the top bar menu and the filter pills, under `Subscriptions`
4. **Verify** it works as designed

-----

## Regression Notes

1. Potential unintended areas of impact

    - UI regressions in the Reader top bar menu.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing.

5. What automated tests I added (or what prevented me from doing so)

    - Updated broken unit tests.

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
